docs/misc.rst

```diff
@@ -0,0 +1,203 @@
+
+
+Notes
+-----
+
+Timing & The Microsoft Azure cloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Timing is a tricky subject when dealing with the Azure cloud as
+some operations can take seconds while others can take up to 20
+minutes in real-world deployments.
+
+Azure APIs make an effort to
+mitigate this by responding to asynchronous operations with a
+"retry after" number to indicate the number of seconds to wait
+before checking the status of an operation again.  Unfortunately,
+this number is more often a woeful underestimate and relying on it
```

Comment:
+this ~~number is more often a woeful underestimate~~ **value is often significantly underestimated** and relying on it